as/cNFT/assembly/mint.ts

```diff
@@ -34,11 +34,14 @@ export function mint(tokenMetadata: TokenMetadata, token_royalty: TokenRoyalty):
     persistent_tokens_royalty.add(tokenId, token_royalty)
 
 
+    token.metadata = tokenMetadata
```

Comment:
What about `TokenRoyalty` here. Shouldn't we add `token_royalty` along with `metadata` here?

Reply:
I'll add it later when I add it first to the the subgraph